packages/cli/README.md

```diff
@@ -19,66 +19,117 @@ This package is available on [npm](http://npmjs.com) as `ai-scan`.
   npm install -g accessibility-insights-scan
 ```
 
-## Example usage
+## Example Usage
 
-if crawling is not enabled (disabled by default), you should provide either --url or --inputFile option, if you provide both, -url only will be used.
+## Single URL Scan
+
+-   You should provide either --url.
+-   An HTML report will be generated in the output folder, previous result for same url will be overwritten.
 
 ```sh
   ai-scan --url https://www.microsoft.com/
 ```
 
+### Options
+
+-   url: --url
+
+```sh
+type: boolean
+describe: The URL to scan for accessibility issues.
+```
+
+-   output: --output
+
+```sh
+type: string
+describe: Output directory. If not set, default is ./ai_scan_cli_output, if you use the same output for different runs, an existing result might be overwritten.
+default: './ai_scan_cli_output'
+```
+
+</br></br>
+
+## Batch Scan
+
+-   You should provide --inputFile option, if you provide both --url and --inputFile, single url scan will be run to scan --url.
+-   A summary HTML report will be generated in the output folder, previous result will be overwritten.
+-   An error log will be generated.
+
 ```sh
   ai-scan --inputFile 'input file path'
 ```
 
-if crawling is not enabled, you should provide either --url or --inputFile option, if you provide both, -url only will be used.
+### Options
+
+-   inputFile: --inputFile
+
+```sh
+type: string
+describe: List of URLs to crawl in addition to URLs discovered from crawling the provided URL.
```

Comment:
suggest updating this to: describe: List of URLs to run automated scans on. If crawling is enabled (disabled by default), the list of URLs to crawl in addition to URLs discovered from crawling the provided URL.